CHANGES.md

```diff
@@ -20,6 +20,7 @@ NoFlo ChangeLog
 * Removed `receiveStreams` option from `WirePattern`
 * Graph JSON schema has been moved to https://github.com/flowbased/fbp, and updated with tests.
 * [babel-core](https://www.npmjs.com/package/babel-core) was removed as a dependency. Install separately for projects needing ES6 component support
+* Added stream helpers to process input. `hasStream` checks if it has a closeBracket for every openBracket, or just data with no brackets, and `getStream` gets them all.
```

Comment:
I would maybe phrase it something like this:

* Added stream helpers for Process API. `hasStream` checks if the input buffer contains a complete stream (matching brackets and data, or only data), `getStream` returns a complete stream of packets. These require `forwardBrackets` to be disabled for the port